### DISCUSSION TITLE
Nomadnet receipt printer, whaaat!!

### DISCUSSION THREAD

**Discussion by kc1awv** [markqvist/Reticulum#357] [Show and tell]
Ok, I decided to have some fun on my once-in-a-lifetime day off. I set up a receipt printer to print messages coming in from Nomadnet. I wonder what else I can do with this, now…

![IMG_0689](https://github.com/markqvist/Reticulum/assets/43918257/363d6632-20fa-4127-8c42-3aa4ca0f828b)

**Comment by faragher** [comment]
You basically made a teletype printer for Reticulum, and that's amazing!

Also glad to see EchoBot is useful.

Edit: First blush, it's great for telemetry, but I like the concept of printing weather reports. You could also use it to text yourself a grocery/shopping list, item by item as you think of them, and then take the physical list when you go to the store.

**Comment by kc1awv** [comment]
EchoBot is certainly useful, when testing this!

I do plan on eventually rolling in a not-Internet based connection to this device and taking it off the RPi 4 for an SBC that's a little less valuable. The original design of this (circa 2017) was for things like printing weather reports, Sudoku puzzles, etc... I even used it as part of the help desk ticketing system at work at one point.

For now, when people send a message to it, it will print it. Limited only by the amount of paper on a roll, really. There should be an announce from the printer somewhere, or if you're into manually typing in addresses, send something to lxmf 50a42aa90131e3750198b2a3ea339b9b

It's all just a fun distraction for the moment until something practical comes out of it 😄

**Comment by kc1awv** [comment]
🤣

![IMG_0691](https://github.com/markqvist/Reticulum/assets/43918257/a5eaa75c-2947-4555-8c4b-362dba46b0aa)

**Comment by kc1awv** [comment]
Y’all are too funny 😂 ![image](https://github.com/markqvist/Reticulum/assets/43918257/1f2666ac-e27a-48cc-ab17-110fe1c38303)

**Comment by trosel** [comment]
Thermal printer messaging system is a great product in general.

Think of it like like an answering machine, but for text messages. You don't want to ping someone's phone because it's not urgent, you just want to leave a message for when they get home.

I want to get my friends set up on it....

**Reply by worst-git-server** [reply]
that would be fun!

**Comment by jackcox15** [comment]
This is actually a insanely cool idea!!!

**Comment by worst-git-server** [comment]
now we basically got fax over reticulum!
